packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -567,9 +519,24 @@ export class GraphQLApi extends GraphqlApiBase {
     return true;
   }
 
-  private formatOpenIdConnectConfig(
-    config: OpenIdConnectConfig,
-  ): CfnGraphQLApi.OpenIDConnectConfigProperty {
+  private formatLogConfig(config?: LogConfig) {
```

Comment:
rather than `format` how about calling it `setupLogConfig` - ditto for the others
it's doing a little more than just formatting